pkg/ccl/storageccl/external_sst_reader.go

```diff
@@ -84,30 +91,33 @@ func ExternalSSTReader(
 	}
 
 	var reader sstable.ReadableFile = raw
+	var memSize int64
```

Comment:
I'd maybe init this to remoteReaderMemoryOverhead, and then below, in the non-enc branch, add the remoteSSTSuffixCacheSize

Reply:
I re-assigned the memSize in the enc branch since the size of that buffer depends on the format of the encrypted SSTs (either the entire buffer or chunked).

Reply:
sure, I guess my thinking is if decryptingReader _doesn't_ download the whole file (which it shouldn't), then we still have all the usual remoteReaderMemoryOverhead coming from the reader under it. /shrug

Reply:
Gotcha, updated to add.